bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js

```diff
@@ -7,6 +7,8 @@ import { filterFeaturesByPropertyFilter } from '../../../mapping/mapmodule/oskar
 import { cleanFilter } from 'oskari-ui/components/FeatureFilter';
 
 export const FEATUREDATA_DEFAULT_HIDDEN_FIELDS = ['__fid', '__centerX', '__centerY', 'geometry'];
+export const ID_FIELD = '__fid';
```

Comment:
I'm not sure why these need to be separate but we can use the constant to make it clear it's the same field:
```suggestion
export const ID_FIELD = '__fid';
export const FEATUREDATA_DEFAULT_HIDDEN_FIELDS = [ID_FIELD, '__centerX', '__centerY', 'geometry'];
```

Reply:
this was more or less a historical remainder. Got rid of it now.